Arduino/libraries/Capacitive_Fingerprint_Reader/finger.cpp

```diff
@@ -9,16 +9,12 @@ uint8_t finger_TxBuf[9];
 
 uint8_t  Finger_SleepFlag = 0;
 
-
-SoftwareSerial mySerial(10, 11); // RX, TX		
-
-
 /***************************************************************************
 * @brief      initialize the SoftwareSerial to communicate with Fingerprint module
 ****************************************************************************/
 void Finger_SoftwareSerial_Init(void)
 {
-	mySerial.begin(19200);	
+	Serial2.begin(19200);	
```

Comment:
Serial2 refers to TX2/RX2. Same applies for Serial1 and Serial3 for TX1/RX1 and TX3/RX3 respectively.